labonneboite/web/contact_form/views.py

```diff
@@ -88,6 +112,13 @@ def add_identication_data(form):
     form.email.data = request.args['email']
 
 
+def get_action_name():
```

Comment:
Je propose de passer request.args en argument, pour que cette fonction ne dépende pas d'un objet statique global et soit plus facile à tester par la suite.

Reply:
will not do it, because if i do it I will want to do it everywhere, for example in `def add_identication_data(form)` and most likely many other places